src/psyclone/domain/gocean/kernel/psyir.py

```diff
@@ -638,7 +712,7 @@ def _validate_stencil(value_list):
                 if not pattern.match(value):
                     raise ValueError(
                         f"Stencil entries should follow the pattern "
```

Comment:
s/pattern/regular expression/ for clarity.

Reply:
Changed.